manual/setup-geth-lodestar.md

```diff
@@ -0,0 +1,95 @@
+Instructions for manually setting up a node on Ephemery using geth and lodestar.
+
+### Prerequisites
+
+[Install golang](https://go.dev/doc/install) for your system.
+[Install Yarn](https://classic.yarnpkg.com/lang/en/docs/install/).
+Check that your [Node](https://nodejs.org/) version is >= v18.15.0.
+
+Then:
+```
+sudo apt install -y git default-jre make gcc
+```
+
+### Download config
+
+```
+mkdir testnet-all
+cd testnet-all
+```
+
+Download the `testnet-all.tar.gz` file for the [latest release of the Ephemery testnet](https://github.com/ephemery-testnet/ephemery-genesis/releases) to this directory using `wget`. Then unzip inside this folder.
+```
+tar -xzf testnet-all.tar.gz
+```
+
+### Generate jwt
+
+```
+openssl rand -hex 32 | tr -d "\n" > "/tmp/jwtsecret"
+```
+
+### Execution Layer
+
```

Comment:
I would give user the option whether to compiler or just download a binary. And for compilation, just refer to geth docs

Reply:
Same comment - I'm happy to add this to both files for Geth